license-compliance/gradle-license-report/allowed-licenses.json

```diff
@@ -364,6 +364,9 @@
     {
       "moduleLicense": "^LGPL-2\\.1$"
     },
+    {
+      "moduleLicense": "^LGPL-2\\.1+$"
```

Comment:
issue: You probably should escape `+` character in the regex
note: this pattern can be joined with the pattern above `^LGPL-2\\.1\\+?$`